cmd/thanos/main_test.go

```diff
@@ -0,0 +1,76 @@
+package main
```

Comment:
Do we need this file? You already have some nice tests in `pkg/prober`

Reply:
Well.. it tests the `metricHTTPListenGroup` (now containing only metrics and profiling)
Those are nowhere else tested. But you find it redundant I can remove it.

Reply:
Maybe they make sense (: